.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+Please make sure that the PR fulfills these requirements
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I checked that there aren't other open Pull Requests for the same update/change.
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
+
```

Comment:
- PR contains a single rule change
- Rule changes includes a comment to describe the reasoning behind the change